Desktop/Application/MaxMix/Services/Communication/MessageSettings.cs

```diff
@@ -26,13 +27,15 @@ public MessageSettings(bool displayNewSession, bool sleepWhenInactive, int sleep
         public bool _sleepWhenInactive;
         public int _sleepAfterSeconds;
```

Comment:
We should probably do a pass on the types and ensure they are the correct expected size for the msg.